src/api/swg-client-event-manager-api.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AnalyticsEvent,EventOriginator} from '../proto/api_messages';
+
+/** @enum {number}  */
+export const FilterResult = {
+  /** the event is allowed to proceed to the listeners */
+  PROCESS_EVENT: 0,
```

Comment:
Is this meant to be ever used by the `swg.js` clients (e.g. non-AMP)? If so, we usually prefer string consts for these, since in most of cases, enums are not used by integrators.

Reply:
It should be used internally by SwG and externally by AMP only.  No publisher should ever access the event manager directly.